llvm/test/CodeGen/SPIRV/llvm-intrinsics/constrained-convert.ll

```diff
@@ -0,0 +1,181 @@
+; RUN: llc -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -mtriple=spirv64-unknown-unknown %s -o - -filetype=obj | spirv-val %}
+
+; CHECK-DAG: OpName %[[#sf:]] "conv"
+; CHECK-DAG: OpName %[[#sf1:]] "conv1"
+; CHECK-DAG: OpName %[[#sf2:]] "conv2"
+; CHECK-DAG: OpName %[[#sf3:]] "conv3"
+; CHECK-DAG: OpName %[[#sf4:]] "conv4"
+; CHECK-DAG: OpName %[[#uf1:]] "conv5"
+; CHECK-DAG: OpName %[[#uf2:]] "conv6"
+; CHECK-DAG: OpName %[[#uf3:]] "conv7"
+; CHECK-DAG: OpName %[[#uf4:]] "conv8"
+; CHECK-DAG: OpName %[[#uf5:]] "conv9"
+; CHECK-DAG: OpName %[[#fs1:]] "conv10"
+; CHECK-DAG: OpName %[[#fs2:]] "conv11"
+; CHECK-DAG: OpName %[[#fs3:]] "conv12"
+; CHECK-DAG: OpName %[[#fs4:]] "conv13"
+; CHECK-DAG: OpName %[[#fs5:]] "conv14"
+; CHECK-DAG: OpName %[[#fu1:]] "conv15"
+; CHECK-DAG: OpName %[[#fu2:]] "conv16"
+; CHECK-DAG: OpName %[[#fu3:]] "conv17"
+; CHECK-DAG: OpName %[[#fu4:]] "conv18"
+; CHECK-DAG: OpName %[[#fu5:]] "conv19"
+; CHECK-DAG: OpName %[[#fe1:]] "conv20"
+; CHECK-DAG: OpName %[[#fe2:]] "conv21"
+; CHECK-DAG: OpName %[[#ft1:]] "conv22"
+; CHECK-DAG: OpName %[[#ft2:]] "conv23"
+
+; CHECK-DAG: OpConvertSToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertSToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertSToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertSToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertSToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertUToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertUToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertUToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertUToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertUToF %[[#]] %[[#]]
+; CHECK-DAG: OpConvertFToS %[[#]] %[[#]]
+; CHECK-DAG: OpConvertFToS %[[#]] %[[#]]
+; CHECK-DAG: OpConvertFToS %[[#]] %[[#]]
+; CHECK-DAG: OpConvertFToS %[[#]] %[[#]]
+; CHECK-DAG: OpConvertFToS %[[#]] %[[#]]
+; CHECK-DAG: OpFConvert %[[#]] %[[#]]
+; CHECK-DAG: OpFConvert %[[#]] %[[#]]
+; CHECK-DAG: OpFConvert %[[#]] %[[#]]
+; CHECK-DAG: OpFConvert %[[#]] %[[#]]
+
+; CHECK-DAG: OpDecorate %[[#sf]] FPRoundingMode RTE
```

Comment:
Thes should have a definitive order, and ideally the checks would be in the function they are testing. Can you just use update_llc_test_checks